src/__stories__/button-story.tsx

```diff
@@ -257,6 +257,11 @@ export const LoadingState: StoryComponent = () => {
             setIsLoading(false);
         }, 2000);
     };
+
+    const handlePressWithPromise = async () => {
+        await new Promise((resolve) => setTimeout(resolve, 3000));
+    };
```

Comment:
You don't need the `async`/`await`
```suggestion
    const handlePressWithPromise = () =>
        new Promise((resolve) => setTimeout(resolve, 3000));
```